competition-simulator.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is different to the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulation is based in [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.5.
```

Comment:
We should specify what version of Python the competition will be run on.

Reply:
I'm not sure we can commit to that quite yet.